scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -525,9 +527,55 @@ task LoadTable {
     fi
 
     if [ $NUM_FILES -gt 0 ]; then
-        bq load --location=US --project_id=~{project_id} --skip_leading_rows=1 --source_format=CSV -F "\t" $TABLE $DIR$FILES ~{schema} || exit 1
-        echo "ingested ${FILES} file from $DIR into table $TABLE"
-        gsutil -m mv $DIR$FILES ${DIR}done/
+        # get list of of pet files and their byte sizes
+        echo "Getting load file sizes(bytes) and path to each file."
+        gsutil du "${DIR}${FILES}" | tr " " "\t" | tr -s "\t" > ~{datatype}_du.txt
+
+        # get total memory in bytes
+        echo "Calculating total files' size(bytes)."
+        TOTAL_FILE_SIZE=$(awk '{print $1}' OFS="\t" ~{datatype}_du.txt| paste -sd+ - | bc)
```

Comment:
FWIW you can also do sum with.... 

```
TOTAL_FILE_SIZE=$( awk '{sum+=$1;} END{print sum;}' ~{datatype}_du.txt)
```

Reply:
I can't recall exactly why now, but I had that in there first and then swapped it for some reason..